src/jshint.js

```diff
@@ -3413,88 +3409,31 @@ var JSHINT = (function() {
     });
   }
 
-  var conststatement = stmt("const", function(context) {
+  function variableStatement(type, statement, context) {
```

Comment:
> Let's run with this approach for now.

Which one? I've already applied @caitp suggestion..  So keep as lexical / variable or combine?